packages/web/src/components/common/wallet-connector-menu/WalletConnectorMenu.tsx

```diff
@@ -92,17 +130,38 @@ const WalletConnectorMenu: React.FC<WalletConnectorMenuProps> = ({
       {connected ? (
         <div className="button-container">
           <MenuHeader>
-            <IconAdenaLogo />
+            {account && account.chainId !== "dev.gnoswap" ? (
+              <IconFailed className="fail-icon" />
+            ) : (
+              <IconAdenaLogo />
+            )}
             <span className="user-address">
               {formatAddress(account?.address || "")}
             </span>
             <IconButtonMaker
               copyClick={copyClick}
               openLinkClick={openLinkClick}
+              themeKey={themeKey}
+              copied={copied}
               onClickDisconnect={onClickDisconnect}
             />
           </MenuHeader>
-          <AmountInfoBox>{balanceText}</AmountInfoBox>
+          {account && account.chainId !== "dev.gnoswap" ? (
```

Comment:
this chainid should be get from environment variable `NEXT_PUBLIC_DEFAULT_CHAIN_ID`.

Reply:
It's okay now